packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -30,6 +30,18 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     ImagePickerPlatform.instance = ImagePickerPlugin();
   }
 
+  /// Returns a [PickedFile] with the image that was picked.
+  ///
+  /// The `source` argument controls where the image comes from. This can
+  /// be either [ImageSource.camera] or [ImageSource.gallery].
+  ///
+  /// Note that the `maxWidth`, `maxHeight` and `imageQuality` arguments are not supported on the web.
```

Comment:
nits: Can we be more explicit and say what happens if `maxWidth` is specified? Maybe like:
```
Any value of `maxWidth`, `maxHeight` or `imageQuality` is ignored.
```

Reply:
You got it!